content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -7,7 +7,7 @@ min-kubernetes-server-version: v1.25
 ---
 
 <!-- overview -->
-{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.30" state="beta" >}}
```

Comment:
- Is line 6 still accurate (you need v1.25 or later)?
- Is line 43 still accurate (you need to enable a feature gate)?

Reply:
Yes to both